### ISSUE TITLE
About the validation set in the dataset.

### ISSUE THREAD

**Issue by chenaixuepython** [Fat-Hai/DRGCNN#3]
Hello. Excuse me. The paper says, “This comprehensive dataset comprises 88,702 fundus images and is thoughtfully partitioned into train/valid/test: 35,126/10,906/42,670.” Why is that? Because the EyePACS dataset I downloaded is partitioned into train/test: 35,126/53,576.
